db/models.py

```diff
@@ -25,6 +30,11 @@ class Movie(models.Model):
     def __str__(self) -> str:
         return self.title
 
+    class Meta:
```

Comment:
The `class Meta` should be defined after all attributes of the `Movie` class, but it is correctly placed here. Ensure this practice is followed consistently across all models.